cmake/FindRdKafka.cmake

```diff
@@ -1,6 +1,7 @@
 # Override default CMAKE_FIND_LIBRARY_SUFFIXES
 if (CPPKAFKA_RDKAFKA_STATIC_LIB)
     if (MSVC)
+        add_definitions("-DLIBRDKAFKA_STATICLIB")
```

Comment:
I don't think this is the best place to put this. Can you move this into the main CMakeLists.txt file? This is supposed to find rdkafka, I wouldn't expect calling "find rd kafka" to add a definition under the hood.

Reply:
Ok.